tendrl/monitoring_integration/graphite/__init__.py

```diff
@@ -474,7 +487,7 @@ def set_brick_count(self, cluster_data):
 
     def set_brick_path(self, cluster_data):
         for cluster in cluster_data:
-            for brick in cluster.details["Brick"]:
+            for brick in cluster["Brick"]:
```

Comment:
Are we maintaining the volumes/bricks etc per cluster basis. I feel the overall structure should be something like

```
clusters = {
    'int_id1': {
        'volumes': [],
        'bricks': []
    }
    'int_id2': {
        'volumes': [],
        'bricks': []
    }
}
```

Reply:
it is clusterwise only,  the current structure is like

    [
        {"integration_id": id,
          "volumes": [],
          "bricks": [],
          .
          .
          .
        },
        {"integration_id": id,
          "volumes": [],
          "bricks": [],
          .
          .
          .
        },
    ]

Reply:
ok